README.md

```diff
@@ -1,4 +1,4 @@
-# kin-ecosystem-android-sample-app
+# kin-ecosystem-android-sdk
 A simple sample app that shows how to initiate  and launch the Kin ecosystem marketplace
```

Comment:
Also the description - 
The kin-ecosystem-android-sdk provide a quick and easy integration with Kin ecosystem. It provide out of the box integration with Kin token on top of stellar blockchain. The SDK create Kin wallet and blockchain account for the end users and a rich marketplace where users can earn and spend Kin.

The  'app' module shows how to initiate  and launch the Kin ecosystem marketplace within digital service app.

Reply:
👍